test/unit/form-param.test.js

```diff
@@ -22,4 +22,27 @@ describe('FormParam', function () {
             expect(fp.toString()).to.be('foo=bar');
         });
     });
+
+    describe('contentType', function () {
+        it('should parse contentType properly', function () {
+            var fp = new FormParam({
+                key: 'fileName',
+                src: 'fileSrc',
+                contentType: 'text/csv',
+                type: 'file'
+            });
+            expect(fp.toJSON()).to.eql({
+                key: 'fileName',
+                value: '',
+                src: 'fileSrc',
+                contentType: 'text/csv',
```

Comment:
The test should only check for the existence of `contentType` in the JSON result.